source/renderer/app/components/widgets/tooltips/QuestionMarkTooltip.js

```diff
@@ -0,0 +1,21 @@
+// @flow
```

Comment:
I am wondering whether such component is needed. As I understand you intended to make it a generic component to be reused in other places. If that is the case IMO in order to do that first we should have a code duplication so that there will be a clear need for such component and we will be able to design its generic api being aware of its usage. I am worried that this component would be used just once.

Moreover I think it is actually simple enough to be inlined.